docs/_source/getting_started/installation/deployments/google-colab.md

```diff
@@ -1,6 +1,7 @@
 # Google Colab
 Google Colab is a free tool that allows users to run and share Jupyter notebooks on Google's servers. It can be useful for quickly trying out models and using GPUs for training and prediction tasks. In the context of Argilla, it can be used for experimenting with pre-trained models, active learning, and computing text embeddings.
 
-To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to read the [cloud providers guide](cloud_providers.md).
+You can run all Argilla tutorials using the Open in Colab button at the top of each tutorial page. To use Argilla with Google Colab, the recommended method is to use Hugging Face Spaces, as [described in this guide](huggingface-spaces.md). This is the easiest way to set up Argilla for use with Google Colab. For other installations, we recommend you to check the [deployments guide](cloud_providers.md).
+
+If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer(../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb).
```

Comment:
```suggestion
If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer(../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb)].
```

Reply:
💯  good catch @tomaarsen:

it should be:

If you want to deploy and run Argilla entirely on Google Colab, check [this tutorial by Moritz Laurer](../../../tutorials/notebooks/deploying-textclassification-colab-activelearning.ipynb)].

Right?

Reply:
👍